src/geometryUtils.jl

```diff
@@ -0,0 +1,119 @@
+const WGS84_SEMI_MAJOR_AXIS = 6378137.0
```

Comment:
would it be better to have a .jl file with all  defined constants. Easier to know what is defined, and what is not?

Reply:
I think it is fine that it is placed here when only this file uses it. It could be a idea later if we have many more const but I think it is overkill for now.